Makefile

```diff
@@ -0,0 +1,26 @@
+BUILD_DIR := build
+MKDIR_P = mkdir -p
+RM = rm -rf
+
+default: build-all
+
+load-deps:
+	${MKDIR_P} ${CURDIR}/${BUILD_DIR}
+	./pre-build.sh
+	touch $@
+
+%:
+	$(MAKE) load-deps clean-up
+	./mkimage ${CURDIR}/${BUILD_DIR}/$@.tar $@
+
+build-all: load-deps clean-up build
```

Comment:
The error is because of the `%:` target in the makefile. `sudo make build-all` invokes that target resulting in the command `./mkimage ${CURDIR}/${BUILD_DIR}/build-all.tar build-all`